docs/operator-manual/argocd-cmd-params-cm.yaml

```
@@ -154,6 +154,8 @@ data:
   reposerver.streamed.manifest.max.extracted.size: "1G"
   # Enable git submodule support
   reposerver.enable.git.submodule: "true"
+  # Set managed-by label for every k8s resource generated by argo Current Status: Alpha (Since v2.8.0)
+  reposerver.managed.by.argo: "false"
```

Could this value instead be `managed-by`? Like

```suggestion
  reposerver.managed-by.argo: "false"
```